taxonomies/vitamins.txt

```diff
@@ -80,7 +80,7 @@ bg:Ретинил палмитат
 cs:retinyl-palmitát
 de:Retinylpalmitat
 el:παλμιτική ρετινόλη
-es:palmitato de retinilo
+es:palmitato de retinilo,palmitate
```

Comment:
"palmitate" is English, not Spanish, right?

Reply:
Changed